### DISCUSSION TITLE
AList服务启动失败，如何解决？急急急！

### DISCUSSION THREAD

**Discussion by Makitoid** [AlistGo/alist#3146] [Q&A] [answered]
我重启了AList，然后第二遍打开时就成这样了：
![image](https://user-images.githubusercontent.com/123004192/214539842-f20c59ce-fef0-40da-a3a1-1de756d4361c.png)
这是怎么回事？因为我在用AList分享本地文件，这一下可不得了，各位大佬麻烦看一下，谢谢！在线等

**Comment by gexgd0419** [comment, marked as answer]
`alist server`会尝试在当前工作目录下搜索（如果搜索不到则创建）配置文件。

图中命令提示符的当前工作目录为`C:\Windows\System32`，由于这里不存在配置文件，alist又没有权限在这里创建配置文件，就会导致其报错退出。

如果配置文件等数据与alist.exe在同一个文件夹，可以在末尾附加`--force-bin-dir`参数，强制其使用exe文件所在的文件夹。

**Reply by anwen-anyi** [reply]
感谢大佬科普 `--force-bin-dir参数` 是干嘛用个 我发现有个 `start`启动参数是干嘛的 原来是这样用的

**Comment by Makitoid** [comment]
问题已解决，谢谢！

**Comment by snowfry** [comment]
借楼分享一下吧，我是因为放在了program files文件夹下，而该文件要求权限较高，所以出现了以上报错，更换文件夹后解决了

**Comment by james70s** [comment]
我也碰到类似的问题。在mac上执行： go run main.go server  

系统就停止在init tool SimpleHttp success: ok  了，也没有任何错误和提示，无法访问 0.0.0.0:5244 地址。

请问这是什么问题？

<img width="885" alt="image" src="https://github.com/alist-org/alist/assets/238471/d3410566-c228-4435-9004-e7b30b276e17">
